### PR TITLE
feat: drop Sentry

### PR DESCRIPTION
**Pull request by P0lip**
Needed by https://github.com/stoplightio/platform-internal/issues/16181

### CODE REVIEW

File: src/ErrorBoundary.tsx [hunk 1]
```diff
@@ -1,49 +1,78 @@
...
 import { FallbackComponent } from './FallbackComponent';
-import { ErrorBoundaryProps, FallbackProps } from './types';
+import { ErrorBoundaryProps, ErrorBoundaryState } from './types';
 
-const wrapFallback = (Component: React.ElementType<FallbackProps>): FallbackRender => {
-  return props => (
-    <Component error={props.error} componentStack={props.componentStack} tryRecovering={props.resetError} />
-  );
-};
+export class ErrorBoundary<P extends object = {}> extends React.PureComponent<
```
Comment (billiegoose) [inline src/ErrorBoundary.tsx:7]: CLASS COMPONENTS??? That brings me back. You know they don't even teach those in code bootcamps anymore. 🤣

Comment (P0lip) [reply src/ErrorBoundary.tsx:7]: well, there's no error hook 😆 if you need an error boundary, you're supposed to use a class component

**Review by billiegoose — APPROVED** [review]
sure whatev :)

File: src/ErrorBoundary.tsx (hunk 1 above)
Comment (billiegoose) [reply src/ErrorBoundary.tsx:7]: 22B

**Pull request merged**